test/k8sT/Updates.go

```diff
@@ -23,6 +24,7 @@ import (
 
 	. "github.com/cilium/cilium/test/ginkgo-ext"
 	"github.com/cilium/cilium/test/helpers"
+	"go.uber.org/atomic"
```

Comment:
What's the motivation behind this package? It's quite trivial to use `sync/atomic`.